CRM/Activity/Form/Activity.php

```diff
@@ -1249,4 +1249,9 @@ public function endPostProcess(&$params, &$activity) {
     }
   }
 
+  public function getActivityTypeDisplayLabels() {
```

Comment:
@demeritcowboy can we add a comment block - the comment below can be moved into it